app/views/unattended/provisioning_templates/user_data/preseed_autoinstall_cloud_init.erb

```diff
@@ -41,8 +45,19 @@ autoinstall:
 <% else -%>
       ssh_authorized_keys: []
 <% end -%>
-  keyboard: {layout: us, toggle: null, variant: ''}
+  keyboard:
+<% if host_param('keyboard') -%>
+    layout: <%= host_param('keyboard') %>
+<% else -%>
+    layout: us
+<% end -%>
```

Comment:
can't this be written as:
```suggestion
    layout: <%= host_param('keyboard', 'us') %>
```

Reply:
I didn't know the `host_param` command has an option to return a "default" parameter which can be passed as an argument. I'm also not really sure how it's done. Could you maybe elaborate this a bit:
https://github.com/theforeman/foreman/blob/473b82f91cad64bd2adede0c8328df2f71f2b5a0/app/models/concerns/host_params.rb#L44-L58

Shouldn't we use it as it is done in the apipie example? By using the `|| <default_param>` syntax?

Reply:
And I was going by https://github.com/theforeman/foreman/blob/3750d2ce8a3609ae1e18d72eedab33d719219e60/app/services/foreman/renderer/scope/macros/host_template.rb#L31-L41

Yours is `@host.host_param` (with `@host` being some `Host` object), mine is a template helper ;)